### DISCUSSION TITLE
Hiking the John Muir Trail! Back in September.

### DISCUSSION THREAD

**Discussion by charliewhitfield** [ivoyager/forum#16] [General]
![](https://www.ivoyager.dev/wp-content/uploads/2024/07/jmt-milkyway.jpg)
Photo from my JMT hike in 2015. More photos [here](https://www.charliewhitfield.photography/John-Muir-Trail).

It's 220 miles with no cell phone coverage almost the entire length. I never even cross a road. Hard to believe we still have places like this!

When I'm back in September, I expect to finally fix the [shadows issue](https://github.com/ivoyager/ivoyager_core/issues/1) with much help from @AlanTheSane. It looks we might have a handle on this. I guess Godot 4.3 might be out then. It should fix many of the web-export problems that came with 4.x. So just maybe we'll move on to our beta and official release finally.

**Comment by AlanTheSane** [comment]
Incredible. Enjoy your trip.

Happy to help, looking at the lighting for this and other limitations you've run into has been extremely informative for myself too.
